src/setuptools_pyproject_migration/__init__.py

```diff
@@ -54,6 +55,12 @@ def initialize_options(self):
     def finalize_options(self):
         pass
 
+    @staticmethod
+    def _transform_contributors(name_string: Optional[str], email_string: Optional[str]) -> List[Contributor]:
+        names = map(str.strip, (name_string or "").split(","))
+        emails = map(str.strip, (email_string or "").split(","))
+        return [{"name": n, "email": e} for n, e in itertools.zip_longest(names, emails, fillvalue="") if n or e]
```

Comment:
Learning so many little tricks with this project. :-)

Reply:
> Learning so many little tricks with this project. :-)

It is definitely surprisingly deep, especially being meant for such a simple task!